### DISCUSSION TITLE
Why did my pull shark badge disappear :(?

### DISCUSSION THREAD

**Discussion by gregdrizz** [community/community#73377] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have contributed to a public repository, it's still public and i got couple of pr's merged there,.
my pull shark badge has disappeared for some reason, i still appear under the "Contributors" there, what may be the reason? 

update: i see it in: 
https://github.com/gregdrizz?tab=achievements

but not in the profile, although i changed the settings for it.

**Comment by 0x48piraj** [comment, marked as answer]
Since the Achievements are still in Beta mode, there may be some issues and bugs. It is possible that the achievements aren't shown in some users.

Try getting other achievements like Yolo and Quickdraw (if you haven't already).

Also, try hiding and unhiding the **Pull Shark** badge.

**Reply by gregdrizz** [reply]
Thanks! Seems like it worked :)
